_docs/Threat-Model.md

```diff
@@ -1,70 +1,126 @@
 ---
 title: Threat Model
 ---
-## Scope of the Threat Model
+#Threat Model
+This model shall provide an overview on the system architecture regarding security. It shall list all attackers and their capabilities. It also shall outline the targeted security goals by describing how they are reached.
```

Comment:
I don't know if this really "[...] shall list all attackers [...]" because we define our attackers in our threat model as we wish because this is how a threat model works. We cannot and do not "list all attackers".